docs/standard/serialization/system-text-json/source-generation-modes.md

```diff
@@ -99,30 +43,37 @@ For information about other known issues with source generation, see the [GitHub
 
-The optimized code doesn't support all of the serialization features that `JsonSerializer` supports. The serializer detects whether the optimized code can be used and falls back to default serialization code if unsupported options are specified. For example, <xref:System.Text.Json.Serialization.JsonNumberHandling.AllowReadingFromString?displayProperty=nameWithType> is not applicable to writing, so specifying this option doesn't cause a fall-back to default code.
+The optimized code doesn't support all of the serialization features that `JsonSerializer` supports. The serializer detects whether the optimized code can be used and falls back to default serialization code if unsupported options are specified. For example, <xref:System.Text.Json.Serialization.JsonNumberHandling.AllowReadingFromString?displayProperty=nameWithType> isn't applicable to writing, so specifying this option doesn't cause a fallback to default code.
 
 The following table shows which options in `JsonSerializerOptions` are supported by the optimized serialization code:
 
 | Serialization option                                                   | Supported by optimized code |
 |------------------------------------------------------------------------|-----------------------------|
+| <xref:System.Text.Json.JsonSerializerOptions.AllowTrailingCommas>      | ❌                          |
 | <xref:System.Text.Json.JsonSerializerOptions.Converters>               | ❌                          |
+| <xref:System.Text.Json.JsonSerializerOptions.DefaultBufferSize>        | ❌                          |
 | <xref:System.Text.Json.JsonSerializerOptions.DefaultIgnoreCondition>   | ✔️                         |
 | <xref:System.Text.Json.JsonSerializerOptions.DictionaryKeyPolicy>      | ❌                          |
 | <xref:System.Text.Json.JsonSerializerOptions.Encoder>                  | ❌                          |
 | <xref:System.Text.Json.JsonSerializerOptions.IgnoreNullValues>         | ❌                          |
 | <xref:System.Text.Json.JsonSerializerOptions.IgnoreReadOnlyFields>     | ✔️                         |
 | <xref:System.Text.Json.JsonSerializerOptions.IgnoreReadOnlyProperties> | ✔️                         |
 | <xref:System.Text.Json.JsonSerializerOptions.IncludeFields>            | ✔️                         |
+| <xref:System.Text.Json.JsonSerializerOptions.MaxDepth>                 | ❌                          |
 | <xref:System.Text.Json.JsonSerializerOptions.NumberHandling>           | ❌                          |
+| <xref:System.Text.Json.JsonSerializerOptions.PropertyNameCaseInsensitive> | ❌                       |
```

Comment:
This is a deserialization-only setting. I'm not sure if it makes sense to include when discussing fast-path serialization, but perhaps it's helpful in emphasizing that there's not fast-path deserialization.